src/bfb_tool_raw_img.centos.sh

```diff
@@ -0,0 +1,330 @@
+#!/bin/bash
+
```

Comment:
Can you please explain what was missing from the last time we tried to create a VM image based on Centos BFB?